addon/components/dynamic-form.js

```diff
@@ -21,12 +22,45 @@ const DynamicForm = Ember.Component.extend({
 
   didReceiveAttrs() {
     let schemaObj = this._initSchema(this.get('schema'));
-    const schemaWithActions = this._addActions(schemaObj);
+    const schemaWithPostRender = this._buildPostRender(schemaObj);
+    const schemaWithActions = this._addActions(schemaWithPostRender);
     const filteredSchema = this._processFilters(schemaWithActions);
     const mappedSchema = this._replaceKeywordsWithFunctions(filteredSchema);
     this.set('renderSchema', mappedSchema);
   },
 
+  _buildPostRender(schemaObj) {
+    let postRenderFns = [];
+    if (this.get('postRender')) {
+      postRenderFns.push(this.get('postRender'));
+    }
+    if (this.get('changeAction')) {
+      let fields = _.reduce(schemaObj.schema.properties, (result, value, key) => {
```

Comment:
Please pardon me if I'm missing something obvious, but could you use 
`let fields = Ember.keys(schemaObj.schema.properties)` 
here?